src/display.jl

```diff
@@ -133,6 +146,9 @@ function subscriptify(n::Int)
 end
 
 
+# fall-back:
```

Comment:
Why do you need `basic_representation` *and* `representation`, or isn't it enough with one function and several methods?